### DISCUSSION TITLE
Loading data from PostGIS

### DISCUSSION THREAD

**Discussion by zacdezgeo** [opengeos/leafmap#14] [Q&A] [answered]
Hi @giswqs, thank you so much for sharing this repo!

I was wondering if you had any recommendations for conveniently loading data from PostGIS. Are there plans for some sort of integration?

**Comment by giswqs** [comment]
Yes, it is possible through GeoPandas. I will add a function to leafmap for this. See https://github.com/giswqs/leafmap/issues/15

https://geopandas.readthedocs.io/en/latest/docs/reference/api/geopandas.GeoDataFrame.from_postgis.html

**Comment by giswqs** [comment, marked as answer]
The feature request for reading data from PostGIS has been implemented. See notebook example
https://github.com/giswqs/leafmap/blob/master/examples/notebooks/14_postgis.ipynb

![Peek 2021-06-01 00-36](https://user-images.githubusercontent.com/5016453/120267548-2a7fa200-c272-11eb-9bfc-30554e3f3a3a.gif)
